test/smtp.test.lua

```diff
@@ -10,6 +10,22 @@ local os = require('os')
 test:plan(1)
 mails = fiber.channel(100)
 
+function write_reply_code(s, l)
+    if l:find('3xx') then
+        s:write('354 Start mail input\r\n')
+    elseif l:find('4xx') then
+        s:write('421 Service not available, closing transmission channel\r\n')
+    elseif l:find('5xx') then
+        s:write('510 Bad email address\r\n')
+    elseif l:find('breakconnect') then
+        s:write('1000 UNKNOWN ERROR\r\n')
+        return -1
```

Comment:
The idea is to test an unexpected reply code or the connection break situation? Now it looks as a strange mix.

Reply:
Fixed.

Reply:
Why it wrotes 'Breaking connection\r\n' to the client? Shouldn't it just break the connection?

Reply:
Yes sure, that sounds more correctly. Fixed.

Reply:
I still see `s:write('Break the connection\r\n')` here. Are we understood each other?

Reply:
Sorry, firstly not, but now i got it.

Reply:
Looks ok now.